### DISCUSSION TITLE
Better translation for CN-document

### DISCUSSION THREAD

**Discussion by JackChanel** [vuejs/pinia#2161] [Ideas]
I have found that there are many translation deviations in the use of documents.
For example, the following link [在 setup() 之外使用存储](https://pinia.web3doc.top/ssr/#%E5%9C%A8-setup-%E4%B9%8B%E5%A4%96%E4%BD%BF%E7%94%A8%E5%AD%98%E5%82%A8)
如果您需要在其他地方使用商店，则需要将 pinia 实例 [已传递给应用程序](https://pinia.web3doc.top/ssr/#install-the-plugin) 传递给 useStore() 函数调用：
should be
如果您需要在其他地方使用store，则需要将 pinia 实例 [已传递给应用程序](https://pinia.web3doc.top/ssr/#install-the-plugin) 传递给 useStore() 函数调用：

**Comment by posva** [comment]
Please do suggestions to docs through PRs 😄
